src/dotnet/Properties/AssemblyInfo.cs

```diff
@@ -3,3 +3,4 @@
 
 [assembly: AssemblyMetadataAttribute("Serviceable", "True")]
 [assembly: InternalsVisibleTo("dotnet.Tests")]
+[assembly: InternalsVisibleTo("Microsoft.DotNet.ProjectJsonMigration.Tests")]
```

Comment:
This feels wrong to me.  Microsoft.DotNet.ProjectJsonMigration.Tests should be testing Microsoft.DotNet.ProjectJsonMigration, not things that are in `dotnet.dll`.